packages/experimental/src/components/APIKeyModal/APIKeyModal.stories.js

```diff
@@ -48,8 +48,33 @@ const defaultProps = {
   successHeader: 'API key successully created',
 };
 
-const Template = (args) => {
-  return <APIKeyModal {...args} />;
+const timeout = () => new Promise((resolve) => setTimeout(resolve, 2000));
```

Comment:
Can we define this timeout somewhere higher and keep it more global than hardcoded on this line?